app/Models/Contact/Document.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Models\Contact;
+
+use App\Models\Account\Account;
+use Illuminate\Support\Facades\Storage;
+use App\Models\ModelBindingHasher as Model;
```

Comment:
@djaiss don't use `ModelBindingHasher` here, unless you send hashid for Documents in vue/axios. Use `ModelBinding` instead, and it's working.

Reply:
Ahhh.